web/controllers/attendance_controller.ex

```diff
@@ -0,0 +1,23 @@
+defmodule CoursePlanner.AttendanceController do
```

Comment:
why do you pass `role: :coordinator`, `role: :teacher` and `role: :student` to the render function?